sdk/core/core/test/inc/az_test_span.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file az_test_span.h
+ *
+ * @brief Test only utilities for helping validate spans.
+ */
+
+#ifndef _az_SPAN_TESTING_H
+#define _az_SPAN_TESTING_H
+
+#include <assert.h>
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <cmocka.h>
+
+#include <az_result.h>
+#include <az_span.h>
+
+#include <stdio.h>
+
+#include <_az_cfg_prefix.h>
+
+
+/**
+ * @brief az_span_for_test_init returns a span over a byte buffer, directly invoking
+ * #az_span_init.  The buffer is initialized with 0xFF to help tests check against buffer overflow.
```

Comment:
comment/source mismatch. Should be 0xCC now.

Reply:
Ugh - checkin, just missed this.  I will fix this up on coming checkin.